tests/make-backup-and-restore-iso/runtest.sh

```diff
@@ -280,8 +282,8 @@ LABEL rear
 
         rlPhaseStartCleanup
             rlFileRestore
-            rlRun "rm -f drive_layout.{old,new}" 0 "Remove lsblk outputs"
-            rlRun "rm -rf /root/rear*.log /var/log/rear/*" 0 "Remove ReaR logs"
+            rlRun "rm -f $REAR_HOME_DIRECTORY/drive_layout.{old,new}" 0 "Remove lsblk outputs"
+            rlRun "rm -rf rear*.log /var/log/rear/*" 0 "Remove ReaR recovery log"
```

Comment:
Will the plain `rear*.log` glob remove correct files? Few lines above, the same file as referenced with the `$REAR_HOME_DIRECTORY` prefix.  (Also, the message change seems unnecessary.)

Reply:
I believe now that `rear*.log` should be always referenced as `/root/rear*.log` and not as `$REAR_HOME_DIRECTORY/rear*.log`. So, few lines above it is wrong. This is because this log is created by ReaR recovery and it creates it under /root unconditionally (more precisely, under `$ROOT_HOME_DIR`, which is set to `~root` by default). So it is not affected by our `$REAR_HOME_DIRECTORY` setting.

Reply:
@lzaoral the last commit should resolve all the `$REAR_HOME_DIRECTORY` confusion ...